src/state/actions/manifest.js

```diff
@@ -58,7 +58,13 @@ export function fetchManifest(manifestId, properties) {
     return fetch(manifestId)
       .then(response => response.json())
       .then(json => dispatch(receiveManifest(manifestId, json)))
-      .catch(error => dispatch(receiveManifestFailure(manifestId, error)));
+      .catch((error) => {
+        if (typeof error === 'object') { // Returned by JSON parse failure
+          dispatch(receiveManifestFailure(manifestId, String(error)));
+        } else {
+          dispatch(receiveManifestFailure(manifestId, error));
```

Comment:
This whole part feels sub-optimal, but wasn't sure about all the scenarios we'll ultimately want to handle.  The existence of the `error` in the manifest state is more or less used as a boolean at this point (since we don't display this error to the user).  The pre-existing behavior was to store the error string (and I think we'll need something more than a true/false state down the road) so I'm casting it as a string for now.

Before this, we were sending a (very odd) error object originating from the JSON library that didn't have any keys or values but would print a string to the error log.  React was choking on this and just throwing an error, and that needed to be addressed for any of this to work.